### ISSUE TITLE
Kirki 4 is now in Beta!

### ISSUE THREAD

**Issue by mapsteps** [themeum/kirki#2406]
Hey everyone,

David here with Kirki.

I'm happy to let you know that we just got started working on Kirki v4 🎉

I encourage everyone that is using Kirki to sign up for our email list.
This will help us to reach out to all of you and let you know when Kirki 4 is ready for testing. 

You can sign up for our mailing list here - (form in the footer or sidebar) https://kirki.org/blog/the-current-state-of-kirki/.
While you are on there, make sure to also read the article if you haven't already :)

Please keep in mind that support for Kirki 3 (GitHub & WordPress forum) is still currently on hold.
New tickets & issues are ignored until we enter the testing phase for Kirki 4. This will help us to maintain focus and minimize distraction.

Once Kirki 4 is ready to be tested, we will send out an email newsletter to all of you.
There will be a 1-month testing & fixing phase after we believe Kirki 4 is ready to be released to make sure everything is running smoothly for everyone. The time period might be extended of course if we run into any issues during public testing.

Thank you.

Best,
David

**Comment by mapsteps** [comment]
Here's a blog post with a few more details :) 

https://kirki.org/blog/kirki-4-is-in-the-works/

**Comment by JimmyAppelt** [comment]
Hi @MapSteps 🙂 

Would it be possible to provide a built package zip in the archive for 4.0 beta's and RC's, so we don't have to build it ourselves?

**Comment by mapsteps** [comment]
Kirki 4 is now in Beta!

Please read the blog post below, test Kirki 4 with your theme and join the discussion in our [Facebook group](https://www.facebook.com/groups/1020376102132534).

https://kirki.org/blog/kirki-4-beta/

**Comment by mapsteps** [comment]
1 to 2 more weeks before we release Kirki 4 to the wild! You can download & test the latest Beta here - https://kirki.org/kirki4-beta3.zip

**Comment by mapsteps** [comment]
MUST READ - Breaking change in Kirki 4 - https://kirki.org/blog/breaking-change-kirki-4/

Kirki 4 will be released soon!
